Makefile

```diff
@@ -0,0 +1,132 @@
+BACKEND_SERVICES ?= autoignore autostarter configuration database detection fileobserver mitigation notifier prefixtree
+TAP_SERVICES ?= bgpstreamhisttap bgpstreamkafkatap bgpstreamlivetap exabgptap riperistap
+FRONTEND_SERVICES ?= frontend
+
+SERVICES ?= $(BACKEND_SERVICES) + $(TAP_SERVICES) + $(FRONTEND_SERVICES)
+
+PUSH ?= false
+BUILD_TAG ?= latest
+CONTAINER_REPO ?= docker.io/inspiregroup
+
+.PHONY: $(BACKEND_SERVICES)
+$(BACKEND_SERVICES): # build backend container
+$(BACKEND_SERVICES):
+	@echo "Building $@ service for tag $(BUILD_TAG)"
+	@docker pull $(CONTAINER_REPO)/artemis-$@:latest
+ifneq ($(BUILD_TAG), "latest")
+	@docker pull $(CONTAINER_REPO)/artemis-$@:$(BUILD_TAG)
```

Comment:
same as with the tap services, this command should be robust to missing images (i.e., when the PR first build occurs)